Ginger/GingerCoreNET/ALMLib/Generic/ALMCore.cs

```diff
@@ -244,8 +244,8 @@ public bool ExportBusinessFlowsResultToALM(ObservableList<BusinessFlow> Business
                         {
                             Reporter.ToLog(eLogLevel.DEBUG, $"Executing RunSet Action Publish to ALM for { GingerDicser.GetTermResValue(eTermResKey.BusinessFlow)} {BizFlow.Name}");
                             Reporter.ToStatus(eStatusMsgKey.ExportExecutionDetails, null, BizFlow.Name, "ALM");
-
-                            if (publishToALMConfig.ToAttachActivitiesGroupReport)
+                            
+                            if (BizFlow.ALMTestSetLevel != "RunSet" && publishToALMConfig.ToAttachActivitiesGroupReport)
```

Comment:
add log if we are skipping attach activities group report for RunSet level
Logs should be added only in case of publishToALMConfig.ToAttachActivitiesGroupReport is true

Reply:
done

Reply:
Done

Reply:
done